frontend/src/bundles/common/components/input/input.tsx

```diff
@@ -28,20 +31,24 @@ const Input = <T extends FormValues>({
     required = false,
     placeholder = '',
     icon = 'none',
+    sx = {},
+    value = undefined,
 }: Properties<T>): JSX.Element => {
     const { field, error, isValid } = useFormField({ name });
 
     return (
         <FormControl isInvalid={!isValid} isRequired={required}>
-            <FormLabel htmlFor={name}>{label}</FormLabel>
+            {label && <FormLabel htmlFor={name}>{label}</FormLabel>}
             <Field
                 {...field}
                 id={name}
                 type={type}
                 placeholder={placeholder}
                 error={error}
+                value={value}
```

Comment:
Why do we need to pass value here?

Reply:
@o-nedashkivska that is used to update the input state, cause after form submitted a clear form is executed.